script.module.phate89/lib/phate89lib/rutils.py

```diff
@@ -25,19 +26,17 @@ def setHeader(self, voice, value):
 
     def log(self,msg, level=2):
         if level<=self.LOGLEVEL:
-            if isinstance(msg, str):
-                msg = msg.decode("utf-8", 'ignore')
-            print u"#### {name}: {text} ####".format(name=os.path.basename(__file__),text=msg)
+            print (u"#### {name}: {text} ####".format(name=os.path.basename(__file__),text=msg))
 
-    def createRequest(self,url,params={},post={},stream=False,addDefault=True):
+    def createRequest(self,url,params={},post={},stream=False,addDefault=True, **kwargs):
         if addDefault:
             params.update(self.DEFPARAMS)
         if post:
-            r = self.SESSION.post(url,params=params,data=post,stream=stream)
+            r = self.SESSION.post(url,params=params,data=post,stream=stream,verify=False, **kwargs)
```

Comment:
Why the verify default to false?

Reply:
oops that was for testing. I'll switch back right away